### PR TITLE
[release-v1.10] Add kn-plugin-workflow 0.32-prelease

### PR DESCRIPTION
**Pull request by dsimansk**
The plugin is currently taken from `0.32-prerelease` branch. Unless there are any changes in the plugin code itself. It can be kept as it's now.

@Kaustubh-pande I've added configuration env var hooks for all 3 `Logic/Workflow` build time props. In case we need to replace them. No need to update hardcoded defaults.

/cc @Kaustubh-pande @rhuss

### CODE REVIEW

File: hack/build.sh.d/kn-plugin-workflow.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Env variables can be injected to override values during build phase
+readonly quarkus_version="${LOGIC_QUARKUS_VERSION:-2.13.8.SP2-redhat-00001}"
+readonly quarkus_platform_group_id="${LOGIC_QUARKUS_GROUP:-com.redhat.quarkus.platform}"
+readonly dev_mode_image="${LOGIC_DEV_MODE_IMAGE:-registry.redhat.io/openshift-serverless-1-tech-preview/logic-swf-devmode-rhel8:1.31}"
```
Comment (dsimansk) [inline hack/build.sh.d/kn-plugin-workflow.sh:6]: @Kaustubh-pande FYI, if we need to inject different values during build phase.

Comment (Kaustubh-pande) [reply hack/build.sh.d/kn-plugin-workflow.sh:6]: Does we need to inject dev_mode_image build ref? or its just for the future.

**Comment by dsimansk** [comment]
infra issue.
/retest

File: hack/build.sh.d/kn-plugin-workflow.sh (hunk 1 above)
Comment (dsimansk) [reply hack/build.sh.d/kn-plugin-workflow.sh:6]: So far it was fine with floating tag, for the next release `:1.31. Depends on Logic team if they would like to use digest or not. For the future and our use case,it's also handy, e.g. for patches unrelated to our code like Quarkus version.

**Pull request merged**